pages/recipes/supabase.mdx

```diff
@@ -1,24 +1,157 @@
 import { Callout } from "nextra/components";
 
-# Supabase
+# Overview
+
+This page is broken down into two sections:
+
+- [Seeding a supabase local development stack](/recipes/supabase#seeding-a-supabase-local-development-stack)
+- [Restoring to a supabase local development stack](/recipes/supabase#restoring-to-a-supabase-local-development-stack)
```

Comment:
Maybe we can put it this way:

```md
Snaplet can provide data for your Supabase project!

Pick your own path:

- [Generate data for your Supabase local development stack](/recipes/supabase#seeding-a-supabase-local-development-stack)
- [Restore data from your production Supabase project to your local development stack](/recipes/supabase#restoring-to-a-supabase-local-development-stack)
```

Reply:
Also, remember "Supabase" is a brand name and must always be capitalised.